xds/src/test/java/io/grpc/xds/orca/OrcaServiceImplTest.java

```diff
@@ -147,17 +145,17 @@ public void testRequestIntervalLess() {
     call.request(1);
     OrcaLoadReport expect = OrcaLoadReport.newBuilder().putUtilization("buffer", 0.2).build();
     verify(listener).onMessage(eq(expect));
-    reset(listener);
+    verify(listener).onHeaders(any(Metadata.class));
```

Comment:
Order onHeaders before onMessage, as that is the expected order (yes, we aren't verifying the order here, but it is confusing with it second).